src/Twig/AppExtension.php

```diff
@@ -11,16 +11,99 @@ public function getFunctions()
     {
         return [
             new TwigFunction('alpha', [$this, 'toAlpha']),
+            new TwigFunction('match_coordinate', [$this, 'matchCoordinate']),
+            new TwigFunction('to_array', [$this, 'toArray']),
```

Comment:
To transform a string into an array you don't need a custom twig function, you can use `"mystring"|split(',')`